pkg/oc/cli/cmd/request_project.go

```diff
@@ -129,11 +132,22 @@ func (o *NewProjectOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, a
 
 // Run implements all the necessary functionality for RequestProject.
 func (o *NewProjectOptions) Run() error {
-	// TODO eliminate this when we get better forbidden messages
-	_, err := o.Client.ProjectRequests().List(metav1.ListOptions{})
+	accessReview := &authorizationapi.SelfSubjectAccessReview{
+		Spec: authorizationapi.SelfSubjectAccessReviewSpec{
+			ResourceAttributes: &authorizationapi.ResourceAttributes{
+				Verb:     "create",
+				Group:    "project.openshift.io",
+				Resource: "projectrequests",
+			},
+		},
+	}
+	accessReviewResponse, err := o.SarClient.Create(accessReview)
 	if err != nil {
 		return err
 	}
+	if !accessReviewResponse.Status.Allowed {
+		return errors.New("you may not request a new project via this API")
```

Comment:
@mfojtik this is less informative than we used to receive.  What if you just hit the list endpoint manually?

Also, I'm really hoping this breaks an integration test.

Reply:
@deads2k i'm pretty sure the existing List() will return this error